src/main.ts

```diff
@@ -195,16 +195,18 @@ async function runSwearJar(stringArray: string[]) {
         }
 
         console.log(result);
-     });
-    await addToCsv(CsvColumnName.SwearjarOutput, swearJarArray);
+    });
+
+    return results;
 }
 
 /**
  * A better profanity filter.
- * @param stringArray is the array of test cases to be ran through the filter
+ * @param data is the array of test cases to be ran through the filter
+ * @returns Array of valid or spam results for each test case.
  */
- async function runCensorSensor(stringArray: string[]) {
-    const censorSensorArray = stringArray.map((testCase: string) => {
+ async function runCensorSensor(data: string[]) {
```

Comment:
Should `async` be there even though it isn't being called in `main`? Just to be consistent

Reply:
Good point. I left it there because it wasn't being used but I can change it to be consistent.